Web/leetcode_reminder.5m.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env python3
+
+# <xbar.title>Leetcode Reminder</xbar.title>
+# <xbar.version>v1.1.0</xbar.version>
+# <xbar.author>zihengjackchen</xbar.author>
+# <xbar.author.github>zihengjackchen</xbar.author.github>
+# <xbar.desc>Leetcode reminder and status in your memu bar!</xbar.desc>
+# <xbar.dependencies>python3</xbar.dependencies>
+# <xbar.image>https://raw.githubusercontent.com/zihengjackchen/xbar-scripts/main/leetcode_reminder/demo.png</xbar.image>
+# <xbar.abouturl>https://github.com/zihengjackchen/xbar-scripts/blob/main/leetcode_reminder/README.md</xbar.abouturl>
+
+# Potential alternative way to access API
+# import cloudscraper
+# scraper = requests.create_scraper(
+#   interpreter='nodejs',
+# 	delay=10,
+#   browser={
+#         'browser': 'firefox',
+#         'platform': 'windows',
+#         'mobile': False,
+#         'headless': True  # Set to True if you want to run headless
+#     }
+# )
+
+import requests
+from datetime import datetime
+import os
+
+# Change these before using this script
+# https://github.com/zihengjackchen/xbar-scripts/blob/main/leetcode/README.md
+USERNAME = ""
+LEETCODE_SESSION = ""
+CSRFTOKEN = ""
```

Comment:
These look like good candidates for [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html).